config/vendors.yml

```diff
@@ -0,0 +1,68 @@
+tribal:
+  B84: 'SITS'
+  C10: 'SITS'
+  K12: 'SITS'
+  K60: 'SITS'
+  K84: 'SITS'
+  L24: 'SITS'
+  L46: 'SITS'
+  L68: 'SITS'
+  N36: 'SITS'
+  O33: 'SITS'
+  S21: 'SITS'
+  S64: 'SITS'
+  S72: 'SITS'
+  # N/A: 'SITS' # Suffolk - ACCREDITS PGCE FOR SCITTS
+  # N/A: 'SITS' # Teesside University
+  G50: 'SITS'
+  R12: 'SITS'
+  S18: 'SITS'
+  W20: 'SITS'
+  U80: nil
+  B32: 'SITS'
+  B72: 'SITS'
+  B78: 'SITS'
+  C55: 'SITS'
+  C58: 'SITS'
+  C99: 'SITS'
+  E14: 'SITS'
+  E28: 'SITS'
+  E84: 'SITS'
+  H72: 'SITS'
+  L34: 'SITS'
+  N38: 'SITS'
+  N77: 'SITS'
+  P80: 'Quercus'
```

Comment:
This is an Ellucian product

Reply:
This was the value in the spreadsheet. Doesn't matter now as I've dropped production information from the file.

Reply:
:+1: